packages/host/app/services/realm-info-service.ts

```diff
@@ -14,9 +14,7 @@ export default class RealmInfoService extends Service {
       return this.cachedRealmURLsForFileURL.get(fileURL)!;
     }
 
-    let response = await this.loaderService.loader.fetch(fileURL, {
-      headers: { Accept: SupportedMimeType.CardSource },
-    });
+    let response = await this.loaderService.loader.fetch(fileURL);
```

Comment:
It seems that this was not the issue breaking test altho I thought it was at one point